_pages/policies/employee-resources-policies/deaf-hoh.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Resources for Deaf and Hard of Hearing individuals in TTS
+tags:
+
+---
+
+Tools, services, points of contact, and tips for navigating the resources available to individuals and teams.
+
+## Services provided by GSA
+
+Once GSA has established that a person is Deaf and/or is Hard of Hearing (HoH), they should not have to provide additional medical documentation for assistive technologies or services related to that disability—only any other reasonable accommodations that they might need.  If you receive a request for medical documentation, consult GSA’s [Office of Civil Rights](https://www.gsa.gov/about-us/organization/office-of-civil-rights-overview) to determine whether that request is appropriate.
```

Comment:
For my own understanding, is there a reason Deaf and Hard of Hearing are capitalized?

Reply:
In this case, more about the upper-/lower-case consistency. Throughout the doc, there's a shortened identification "Deaf/HoH," and the lowercased "deaf/hoh" format is a bit too informal. Does this make sense?

Outside of this case, a capitalized 'D' in "Deaf" would signal someone who identifies themselves being within Deaf culture while lower 'd' would signal a general identification with hearing loss.

Hope this helps.

Reply:
It does! Thank you Nick!